### DISCUSSION TITLE
RDF in AIMD is different from RDF in deepmd

### DISCUSSION THREAD

**Discussion by ChenXianSeng1** [deepmodeling/deepmd-kit#2649] [Q&A]
### Summary

The RDF I computed with AIMD in vasp is different from the RDF computed by the model I ran with deppmd.

### DeePMD-kit Version

2.2.2

### TensorFlow Version

2.9.0

### Details

The RDF I computed with AIMD in vasp is different from the RDF computed in lammps for the model I ran with deppmd.
The following is the input file for AIMD：
![AIMD](https://github.com/deepmodeling/deepmd-kit/assets/138181076/d270cf8e-99ad-4b32-9962-abb09d4c031d)
The following is the input file for lammps：
![lammps](https://github.com/deepmodeling/deepmd-kit/assets/138181076/580bba12-40d6-42f0-8c7a-b63298d1366b)
The following is the error analysis of the deepmd model：
![DP DFT](https://github.com/deepmodeling/deepmd-kit/assets/138181076/99d5a0ee-b090-4199-b067-41c2daa941a6)
![lcurve](https://github.com/deepmodeling/deepmd-kit/assets/138181076/69e6bd00-5301-41e1-b88d-bed41e855051)

**Comment by wanghan-iapcm** [comment]
No description of the coverage of data and the MD conditions. We cannot give you suggestion. In general, you may follow https://docs.deepmodeling.com/projects/deepmd/en/master/troubleshooting/precision.html to check the accuracy of your model .
